src/anet.c

```diff
@@ -455,7 +455,7 @@ static int anetTcpGenericConnect(char *err, const char *addr, int port,
          * Make sure connection-intensive things like the benchmark tool
          * will be able to close/open sockets a zillion of times.
          */
-        int sockflags = ANET_SOCKET_CLOEXEC | ANET_SOCKET_REUSEADDR;
+        int sockflags = ANET_SOCKET_CLOEXEC;
```

Comment:
This is inside the function anetTcpGenericConnect(). Are you sure it's only for unix sockets?

Reply:
Oops, updated the wrong line, reverted it.